src/js/components/pages.js

```diff
@@ -224,6 +207,21 @@ export class PersonPage extends Page {
     return pageContent;
   }
 
+  renderPartyAffiliation() {
```

Comment:
this would be even better on CouncillorPage to make it even less likely to accidentally be shown on an AdministratorPage